src/main/kotlin/me/owdding/customscoreboard/feature/customscoreboard/elements/ElementParty.kt

```diff
@@ -10,22 +11,41 @@ import tech.thatgravyboat.skyblockapi.api.profile.party.PartyAPI
 object ElementParty : Element() {
     override fun getDisplay() = buildList {
         val list = PartyAPI.members.distinctBy { it.name }
-        add("§9Party (${list.size})")
+        add("§9Party (${list.size})") {
+            this.hover = listOf("§7Click to view party info")
+            this.serverCommand = "/party list"
+        }
         if (LinesConfig.showPartyLeader) {
             PartyAPI.leader?.let {
-                add("§7- §f${it.name ?: "§cUnknown"} §e♚")
+                addPerson("§7- §f${it.name ?: "§cUnknown"} §e♚", it.name)
             }
         }
 
         list
             .take(LinesConfig.maxPartyMembers)
             .filter { LinesConfig.showPartyLeader && it != PartyAPI.leader }
             .forEach {
-                add("§7- §f${it.name ?: "§cUnknown"}")
+                val line = "§7- §f${it.name ?: "§cUnknown"}"
+                if ((KnownMods.SKYBLOCK_PV.installed || KnownMods.SKYBLOCKER.installed) && it.name != null) {
+                    addPerson(line, it.name)
+                } else add(line) {
+                    this.hover = listOf("§7Click to open SkyCrypt.")
+                    this.link = "https://sky.shiiyu.moe/stats/${it.name}"
+                }
```

Comment:
shouldnt this whole thing be in addPerson? since it should also be done for the leader too